styles/styles.css

```diff
@@ -1295,7 +1295,7 @@ main > .section.small-screen-cta-container > .grid-container {
         inset: -40px -100px auto auto;
       }
 
-      @media (min-width: 105rem) {
+      @media (min-width: 120rem) {
```

Comment:
hmm why this one? so it overflows?

Reply:
This one was so the SVG isn't too far to the left at the breakpoint. I looked at this again though and pushed up another adjustment so the largest breakpoint shifts to the right and I just got rid of this one.